dts/bindings/spi/gaisler,spimctrl.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023 Frontgrade Gaisler AB
+# SPDX-License-Identifier: Apache-2.0
+
+description: GRLIB SPIMCTRL
+
+compatible: "gaisler,spimctrl"
+
+include: spi-controller.yaml
+
+properties:
+  reg:
+    required: true
+
+  mem:
```

Comment:
I could not find where this is used in the driver

Reply:
It was not used and is now removed from the .yaml file.